src/model/engines/engines.nit

```diff
@@ -15,3 +15,4 @@ module engines
 
 import engine_base
 import pep8
+import nitc
```

Comment:
I do not like the activation of the nitc engine in the dev- or prod- server without proper sand-boxing.